examples/timeseries_classification_from_scratch.qmd

```diff
@@ -0,0 +1,173 @@
+---
+title: "Timeseries classification from scratch"
+authors:
+  - "[hfawaz](https://github.com/hfawaz/)"
+  - "[terrytangyuan](https://github.com/terrytangyuan) - R adaptation"
+  - "[t-kalinowski](https://github.com/t-kalinowski) - R adaptation"
+date-created: 2022/12/03
+date-last-modified: 2022/12/03
+description: "Training a timeseries classifier from scratch on the FordA dataset from the UCR/UEA archive."
+categories: [timeseries]
+aliases:
+  - ../guide/keras/examples/timeseries_classification_from_scratch/index.html
+---
+
+## Introduction
+
+This example shows how to do timeseries classification from scratch, starting from raw
+CSV timeseries files on disk. We demonstrate the workflow on the FordA dataset from the
+[UCR/UEA archive](https://www.cs.ucr.edu/%7Eeamonn/time_series_data_2018/).
+
+## Setup
+
+```{r}
+library(tensorflow)
+library(keras)
+set.seed(1234)
+```
+
+```{r, message=FALSE}
+train_df <- readr::read_tsv("https://raw.githubusercontent.com/hfawaz/cd-diagram/master/FordA/FordA_TRAIN.tsv", col_names = FALSE)
```

Comment:
Can you please use `keras::get_file()` here to avoid downloading the dataset on repeated renders?

Something like: 
```
# where the datasets can be accessed
url <- "https://raw.githubusercontent.com/hfawaz/cd-diagram/master/FordA"

train_df <- "FordA_TRAIN.tsv" %>%
  get_file(., file.path(url, .)) %>%
  readr::read_tsv(col_names = FALSE)
```